### DISCUSSION TITLE
Reusable workflows can't be fetched

### DISCUSSION THREAD

**Discussion by aramfe** [community/community#40939] [Actions]
### Select Topic Area

Bug

### Body

There are already several topics regarding this issue already, but because all of them are just hijacking existing threads from several month ago regarding similar problems, I decided to create this thread for the current issue specifically.

Our workflows structure is defined by using several reusable workflows. We call them all locally. It worked fine for several month, but since one or two days ago, we started experiencing blocking issues: We are not able to start any workflow runs at all, because the following parsing error is thrown, whenever we try to start a run: 

<img width="1304" alt="Screenshot 2022-12-06 at 16 34 17" src="https://user-images.githubusercontent.com/85229755/205955151-eb4921bf-e8c8-43c0-a1c8-2f9f3339a944.png">

Since we didn't change anything, we expect that the problem lies by Github itself. We would appreciate a very fast fix or quick solution to this highly critical problem, since we can't deploy at all anymore, as we are heavily relying on Github here.

Thanks in advance!

**Comment by ossanna16** [comment]
Hi there @aramfe and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
